examples/data-objects/focus-tracker/src/FocusTracker.ts

```diff
@@ -134,7 +184,44 @@ export class FocusTracker extends TypedEventEmitter<IFocusTrackerEvents> {
         return statuses.join(newLineSeparator);
     }
 
-    public getPresenceForUser(userId: string, clientId: string): boolean | undefined {
+    /**
+     * Get a copy of the internal mouse position presences map
+     * @returns The map copy
+     */
+
+    public getFocusPresenceForUser(userId: string, clientId: string): boolean | undefined {
         return this.focusMap.get(userId)?.get(clientId);
     }
+
+    /**
+     * Get a copy of the internal mouse position presences map
+     * @returns The map copy
+     */
+    public getMousePresences(): Map<string, Map<string, [number, number]>> {
```

Comment:
I don't see this called anywhere, but it seems probably useful (probably a better method to expose on the `FocusTracker` than `getMousePresencesString`).

Maybe consider removing the `get*String` methods from here, and instead have the view call this method and build its own strings (since the choice to present as a string is a view choice).